mscore/stafftextproperties.cpp

```diff
@@ -184,10 +184,10 @@ StaffTextProperties::StaffTextProperties(const StaffTextBase* st, QWidget* paren
             const Channel* a = part->instrument(tick)->channel(i);
             QTreeWidgetItem* item = new QTreeWidgetItem(channelList);
             item->setData(0, Qt::UserRole, i);
-            if (a->name().isEmpty() || a->name() == Channel::DEFAULT_NAME)
-                  item->setText(0, qApp->translate("channel", Channel::DEFAULT_NAME));
-            else
-                  item->setText(0, qApp->translate("InstrumentsXML", a->name().toUtf8().data()));
+            QString name = a->name();
+            if (a->name().isEmpty())
+                  name == Channel::DEFAULT_NAME;
```

Comment:
This is supposed to be assignment, right?

Reply:
Outch... Of course